requirements_plus.txt

```diff
@@ -7,6 +7,7 @@ geojson>=1.3.2
 folium>=0.2.1
 mplleaflet>=0.0.5
 statsmodels>=0.6.1
+mapclassify
```

Comment:
This is the only dependency I ported from the env files. However, I'm not familiar with the `pysal` code and maybe the tests require more packages here. (Locally I noticed that 49 tests were skipped.)